base/atomics.jl

```diff
@@ -314,17 +327,23 @@ julia> x[]
 """
 function atomic_min! end
 
-unsafe_convert(::Type{Ptr{T}}, x::Atomic{T}) where {T} = convert(Ptr{T}, pointer_from_objref(x))
+unsafe_convert(::Type{Ptr{T}}, x::Atomic{T}) where {T} =
+    convert(Ptr{T}, pointer_from_objref(x))
 setindex!(x::Atomic{T}, v) where {T} = setindex!(x, convert(T, v))
 
 const llvmtypes = IdDict{Any,String}(
-    Bool => "i8",  # julia represents bools with 8-bits for now. # TODO: is this okay?
```

Comment:
are source comments not allowed anymore?

Reply:
it's a bug, yo, and listed in [known issues](https://github.com/domluna/JLFmt.jl#present-limitations)